sdk/go/common/workspace/plugins.go

```diff
@@ -1105,13 +1105,14 @@ func parsePluginSpecFromURL(
 	default:
 		return PluginSpec{}, inference, errors.New(`unknown URL scheme: expected "git" or "https"`)
 	}
-
-	gitURL, _, err := gitutil.ParseGitRepoURL(parsedURL.String())
-	if err != nil {
-		return PluginSpec{}, inference, err
+	urlWithoutAuth := &url.URL{
```

Comment:
Worth leaving a comment that were purposefully dropping auth info

Reply:
Good shout.  Added :heavy_check_mark: